app/mainSqlite/java/org/storymaker/app/SceneEditorActivity.java

```diff
@@ -32,8 +32,6 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
-import net.micode.soundrecorder.SoundRecorder;

```

Comment:
Something odd is happening in your workflow.  this change is already on the dev branch from my commit, why is it showing up as if you made this edit?

https://github.com/StoryMaker/storymaker/commit/acf7649f69f12341db5a052a937a26c3c0de3c0e#diff-d6c548e2dfa8532b6b2ca13c2eaaf763L35